app/assets/stylesheets/styles/components/users/_profile.css.scss

```diff
@@ -0,0 +1,136 @@
+.member-profile {
+
+  ul {
+    list-style: none;
+    margin-left: 0;
+  }
+
+  .member-content {
+    margin-top: 2rem;
+  }
+
+  .edit-profile-button {
+    margin-bottom: 1rem;
+    margin-right: 1rem;
+  }
+
+  .create-guide-button a {
+    color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.